helper-cli/src/main/kotlin/commands/CreateAnalyzerResultFromPackageListCommand.kt

```diff
@@ -51,6 +51,7 @@ import org.ossreviewtoolkit.model.config.ScopeExclude
 import org.ossreviewtoolkit.model.config.ScopeExcludeReason
 import org.ossreviewtoolkit.model.mapper
 import org.ossreviewtoolkit.model.orEmpty
+import org.ossreviewtoolkit.model.utils.toPurl
```

Comment:
Commit message: "properly ~~in~~ via", and maybe say "manually" instead of "properly"?